src/mlpack/methods/ann/layer/alpha_dropout.hpp

```diff
@@ -56,17 +54,33 @@ class AlphaDropout
    * @param ratio The probability of setting a value to alphaDash.
    * @param alphaDash The dropout scaling parameter.
    */
-  AlphaDropout(const double ratio = 0.5,
-               const double alphaDash = -alpha * lambda);
+  AlphaDropoutType(const double ratio = 0.5,
+                   const double alphaDash = -alpha * lambda);
 
   /**
-   * Ordinary feed forward pass of the alpha_dropout layer.
+   * Clone the DropoutType object. This handles polymorphism correctly.
```

Comment:
```suggestion
   * Clone the AlphaDropoutType object. This handles polymorphism correctly.
```

I think technically still correct, since it's a special type of Dropout.